app/helpers/data_getter.py

```diff
@@ -216,16 +216,15 @@ def get_tracks_by_event_id():
         return Track.query.filter_by(event_id=get_event_id())
 
     @staticmethod
-    def get_sessions(event_id, state='accepted'):
```

Comment:
You can keep the parameter with default value as `None`. If it is `None` then show accepted and confirmed. Else show what the parameter says.